src/App/Action/Account/Meetup/EditMeetupAction.php

```diff
@@ -0,0 +1,100 @@
+<?php
+declare(strict_types = 1);
+
+namespace App\Action\Account\Meetup;
+
+use App\Service\Location\FindLocationByUuid;
+use App\Service\Meetup\FindMeetupByUuidInterface;
+use Doctrine\ORM\EntityManagerInterface;
+use Psr\Http\Message\ResponseInterface as Response;
+use Psr\Http\Message\ServerRequestInterface as Request;
+use Ramsey\Uuid\Uuid;
+use Zend\Diactoros\Response\HtmlResponse;
+use Zend\Diactoros\Response\RedirectResponse;
+use Zend\Expressive\Helper\UrlHelper;
+use Zend\Expressive\Template\TemplateRendererInterface;
+use Zend\Form\FormInterface;
+use Zend\Stratigility\MiddlewareInterface;
+
+final class EditMeetupAction implements MiddlewareInterface
+{
+    /**
+     * @var TemplateRendererInterface
+     */
+    private $templateRenderer;
+
+    /**
+     * @var FormInterface
+     */
+    private $form;
+
+    /**
+     * @var EntityManagerInterface
+     */
+    private $entityManager;
+
+    /**
+     * @var FindMeetupByUuidInterface
+     */
+    private $findMeetupByUuid;
+
+    /**
+     * @var FindLocationByUuid
+     */
+    private $findLocationByUuid;
+
+    /**
+     * @var UrlHelper
+     */
+    private $urlHelper;
+
+    public function __construct(
+        TemplateRendererInterface $templateRenderer,
+        FormInterface $form,
+        EntityManagerInterface $entityManager,
+        FindMeetupByUuidInterface $findMeetupByUuid,
+        FindLocationByUuid $findLocationByUuid,
+        UrlHelper $urlHelper
+    ) {
+        $this->templateRenderer = $templateRenderer;
+        $this->form = $form;
+        $this->entityManager = $entityManager;
+        $this->findLocationByUuid = $findLocationByUuid;
+        $this->urlHelper = $urlHelper;
+        $this->findMeetupByUuid = $findMeetupByUuid;
+    }
+
+    public function __invoke(Request $request, Response $response, callable $next = null) : Response
+    {
+        $meetup = $this->findMeetupByUuid->__invoke(Uuid::fromString($request->getAttribute('uuid')));
```

Comment:
Yeah, that's possible, but annoyingly IDE support is shaky for that syntax (it doesn't error as such, type detection doesn't work) :/